compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/awt/ComposePanelTest.kt

```diff
@@ -33,7 +33,6 @@ class ComposePanelTest {
                 frame.contentPane.add(composePanel)
                 frame.isUndecorated = true
 
-                frame.isVisible = true
```

Comment:
Why this change?

Reply:
`pack` doesn't work as expected immediately after `frame.isVisible=true`. The test was failing sometimes here:
```
assertThat(frame.size).isEqualTo(Dimension(234, 345))
```
the actual size was (2,3).

I don't know if the issue in Skiko, or in Swing, but the purpose of the test isn't checking `pack` immediately after `isVisible` (which is very rare use case)